airflow/cli/commands/task_command.py

```diff
@@ -158,33 +158,32 @@ def task_run(args, dag=None):
     # behind multiple open sleeping connections while heartbeating, which could
     # easily exceed the database connection limit when
     # processing hundreds of simultaneous tasks.
-    settings.configure_orm(disable_connection_pool=True)
```

Comment:
This change is needed because I realized test_task_commands sometimes interferes with other tests because this line sets `disable_connection_pool` to True without setting it back. So when test_task_commands run first, connection pool is disabled globally and never restored.